src/views/pages/address/enter-address-form.njk

```diff
@@ -0,0 +1,182 @@
+{% import 'includes/macros.njk' as macros %}
+
+{% set validateNameOrNumber = "validateInput(this, 50, '" + i18n.address.enterAddress.errorMessages.nameOrNumberMissing | escape + "', '" + i18n.address.enterAddress.errorMessages.nameOrNumberLength | escape + "');" %}
+{% set validateAddressLine1 = "validateInput(this, 50, '" + i18n.address.enterAddress.errorMessages.addressLine1Missing | escape + "', '" + i18n.address.enterAddress.errorMessages.addressLine1Length | escape + "');" %}
+{% set validateTownOrCity = "validateInput(this, 50, '" + i18n.address.enterAddress.errorMessages.townOrCityMissing | escape + "', '" + i18n.address.enterAddress.errorMessages.townOrCityLength | escape + "');" %}
+{% set validatePostcode = "validateInput(this, 15, '" + i18n.address.enterAddress.errorMessages.postcodeMissing | escape + "', '" + i18n.address.enterAddress.errorMessages.postcodeLength | escape + "');" %}
+
+{% block content %}
+
+  <form class="form" action={{ props.currentUrl }} method="post">
+    <input type="hidden" name="pageType" value={{ props.pageType }}>      
+
+    {% include "includes/csrf_token.njk" %}
+
+    {% include "includes/errors.njk" %}
+
+    <div>
+
+        <fieldset class="govuk-fieldset">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+            <h1 class="govuk-heading-xl">
+            {{ pageTitle }}
+            </h1>
+        </legend>
+
+        <p class="govuk-hint">{{ addressTitleHint1 }}</p>
+        <p class="govuk-hint">{{ addressTitleHint2 }}</p>
+        <p class="govuk-hint">{{ addressTitleHint3 }}</p>
+        
+        <br>
+
+        {{ govukInput({
+            label: {
+            text: i18n.address.enterAddress.propertyNameOrNumber,
```

Comment:
indentation looks a bit wrong here, thing like the label: { contents should be indented and a few others